### PR TITLE
Repair `torch.nn.functional.gelu` function

### PR DESCRIPTION
**Pull request by suvadityamuk**
Repaired and added functionality for `approximate` kwarg. All tests now pass locally. This function is required in order to transpile some models from `timm`

### CODE REVIEW

File: ivy/functional/frontends/torch/nn/functional/non_linear_activation_functions.py [hunk 1]
```diff
@@ -98,14 +98,11 @@ def softmax(input, dim=None, _stacklevel=3, dtype=None):
...
-    # if approximate == "none":
-    # approximate = False
-    # else:
-    # approximate = True
-    return ivy.gelu(input, approximate=False)
+def gelu(input, approximate=None):
+    if approximate is None:
+        approximate = False
+
+    return ivy.gelu(input, approximate=approximate)
```
Comment (fspyridakos) [inline ivy/functional/frontends/torch/nn/functional/non_linear_activation_functions.py:105]:
```suggestion
def gelu(input, approximate='none'):
    if approximate is 'none':
        return ivy.gelu(input, approximate=False)
    elif approximate is 'tanh':
        return ivy.gelu(input, approximate=True)
    else:
        raise ivy.utils.exceptions.IvyException(
             "approximate argument must be either none or tanh."
        )
```

**Review by fspyridakos — CHANGES_REQUESTED** [review]
I suggested a change to the code, please also add `approximate` to the test since it not being tested currently. Should be `approximate = st.sampled_from(["none", "tanh"])`

File: ivy/functional/frontends/torch/nn/functional/non_linear_activation_functions.py (hunk 1 above)
Comment (suvadityamuk) [reply ivy/functional/frontends/torch/nn/functional/non_linear_activation_functions.py]: Have incorporated this, thank you!

**Comment by fspyridakos** [comment]
LGTM

**Pull request merged**